### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by EvanNelson04**
Readme update looking for review

### CODE REVIEW

**Review by hayleepierce — APPROVED** [review]
This is a good base README.md. We can continue to add to this as we go. Thank you!

File: README.md [hunk 1]
```diff
@@ -20,3 +28,44 @@ developer-development/feedback-overall-course-assessment.yml \
 --github-repository-prefix computer-science-203-fall-2023-course-assessment \
 --transfer-report
 ```
+
+## 🎉 Introduction
+
+- Cellveyor is a python program that produces assignment reports for students or
+classes. Using cellveyor will publicly give grade reports including feedback for
```
Comment (jnormile) [inline README.md:35]: We should be consistent with the casing for **Cellveyor** (either always all lowercase, or always have the "C" capitalized).

File: README.md [hunk 2]
```diff
@@ -20,3 +28,44 @@ developer-development/feedback-overall-course-assessment.yml \
...
+created assignments. Using the cellveyor tool will quickly run and send reports to
+members included in a locally created google spreadsheet. By running the command
+created, this will quickly and automatically send out these reports in a very timely
+and efficient fashion.
+
+## 😂 Definitions
```
Comment (jnormile) [inline README.md:41]: While I see what you're going for based on the `chasten` repository, I'm not sure a "Definitions" section is really warranted. "Cellveyor" isn't necessarily a real word (like chasten); instead, if you're interested in a section like this, re-tool it to give some explanation behind the tool's name.

**Review by hayleepierce — APPROVED** [review]

**Review by jnormile — COMMENTED** [review]
Thanks for putting a README together! I think there's still room to tighten up the documentation--please refer to my comments for opportunities to bring this closer to a professional quality!

File: README.md [hunk 3]
```diff
@@ -20,3 +28,44 @@ developer-development/feedback-overall-course-assessment.yml \
...
+
+## 🔋Features
+
+- 🚀 Fully customizable command line interface
+- ✨ Automated generation of grade-based reports sent to students
+- 🪂 Rich command line interface with many various arguments
+
+## ⚡️ Requirements
+
+- Cellveyor git hub repository
```
[the remark this answers is not in the document]
Comment (EvanNelson04) [reply README.md:61]: fixed

File: README.md (hunk 1 above)
Comment (EvanNelson04) [reply README.md]: I changed the definition of cellveyor a bit, could I use the sentence usage part?

File: README.md (hunk 2 above)
Comment (EvanNelson04) [reply README.md]: I changed the definition of cellveyor a bit, could I use the sentence usage part?

File: README.md (hunk 1 above)
Comment (EvanNelson04) [reply README.md]: okay, I reviewed that and fixed all of it

**Review by jnormile — APPROVED** [review]
Thanks for putting this content together, and quickly implementing feedback! It looks good!

**Review by wolff01 — APPROVED** [review]
This looks really good. This is a huge contribution to the cellveyor program thank you.

**Pull request merged**